src/Github.js

```diff
@@ -267,16 +277,20 @@ class Github {
             }`
         let issues = [];
         // await Promise.all([]);
-        for (const repo of searchRepos) {
+        let resultSets = searchRepos.map(repo => {
             let [owner, project] = repo.split('/');
-            let results = await graphql(query, {
+            return graphql(query, {
                 headers: {
                     authorization: `token ${token}`
                 },
                 owner: owner,
                 project: project,
                 labels: labels
             });
+        });
+        Promise.all(resultSets);
+        for (const resultSet of resultSets) {
+            let results = await resultSet;
```

Comment:
As discussed IRL, future work could make this trigger updates as each query returns, but that's for another day.